veggieseasons/lib/screens/details.dart

```diff
@@ -73,12 +73,9 @@ class DetailsScreen extends StatelessWidget {
           Positioned(
             right: 0.0,
             left: 0.0,
-            child: Hero(
```

Comment:
What's this for? It'd be nice to still do a hero

Reply:
Agreed, but right now, they don't look good enough to keep in a published app. I need to do more research to figure out how to have a larger widget serve as the hero, get the transition to match, and some other details.